src/components/sidekick/index.tsx

```diff
@@ -46,7 +49,7 @@ export class Container extends React.Component<Properties> {
     if (this.isSecondary) {
       return <GroupManagementContainer />;
     } else {
-      return <MessengerList />;
+      return this.props.isUserProfileOpen ? <UserProfileContainer /> : <MessengerList />;
```

Comment:
I will improve the handling of these containers as follow ups when continuing work on users profile